src/routes.rs

```diff
@@ -150,7 +150,7 @@ async fn get_coins(
     amount: u64,
 ) -> Result<Vec<Input>, DispenseError> {
     wallet
-        .get_spendable_resources(*base_asset_id, amount)
+        .get_spendable_resources(*base_asset_id, amount, None)
```

Comment:
what is the last value here?

Reply:
Honestly, not great docs on it, but if it passes the tests...